src/features/user/CompleteSignup/index.tsx

```diff
@@ -165,6 +166,7 @@ export default function CompleteSignup(): ReactElement | null {
 
   const sendRequest = async (bodyToSend: any) => {
     setRequestSent(true);
+    setDisable(true);
```

Comment:
You're setting `disable` to `true`. In case of success, the user will be redirected away from this page, so there isn't any need to set it as  `false`. 

The failure scenario is not handled, however. The button will remain disabled if the request fails, preventing the user from attempting the request again (which is desirable).